### PR TITLE
[Issue 704] Breadcrumbs

### LINKED ISSUE #704: [Task]: Add breadcrumbs component for subpages
### Summary

image

### Acceptance criteria

_No response_

### PR DESCRIPTION
**Pull request by SammySteiner**
## Summary
Fixes #704 

## Changes proposed
Added breadcrumb component, constants, types, story, and test. Added component to process and research page.

## Additional information
Process:
image

Research:
image

Story:
image

### CODE REVIEW

File: frontend/src/constants/breadcrumbs.ts
Comment (andycochran) [inline frontend/src/constants/breadcrumbs.ts:1]: Could we infer hierarchy from routes so we don't have to define the breadcrumb trail for each page? Might there be downsides for doing that? We'd still need a lookup for the title strings. 🤔 Just thinking out loud here. No change required, maybe just food for thought as we get into more complex info architecture and page/route structures.

File: frontend/stories/components/Breadcrumbs.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import Breadcrumbs from "src/components/Breadcrumbs";
+
+const meta: Meta<typeof Breadcrumbs> = {
+  title: "Components/Breadcrumbs",
+  component: Breadcrumbs,
+};
+export default meta;
+
+export const Home = {
```
Comment (andycochran) [inline frontend/stories/components/Breadcrumbs.stories.tsx:12]: I don't think we'd ever show breadcrumbs on root. Okay to show how it'd work if we did tho.

**Review by andycochran — APPROVED** [review]

File: frontend/stories/components/Breadcrumbs.stories.tsx (hunk 1 above)
Comment (SammySteiner) [reply frontend/stories/components/Breadcrumbs.stories.tsx:12]: Yeah, I wasn't sure what to use for the default story. I could change it to be either Research or Process 🤷, let me know if you have a preference

Comment (andycochran) [reply frontend/stories/components/Breadcrumbs.stories.tsx:12]: Probably fine as-is. We can revisit in the future as our route/page requirements become more complex.

**Review by acouch — APPROVED** [review]
LGTM!

**Pull request merged**